src/process_utility.h

```diff
@@ -24,7 +24,11 @@ typedef struct ProcessUtilityArgs
 	ParamListInfo params;
 	DestReceiver *dest;
 	List *hypertable_list;
+#if PG13_GE
+	QueryCompletion *qc;
```

Comment:
Could you use a more descriptive name , instead of qc ?  Maybe query_completion ?

Reply:
changed